internal/pkg/cli/local_run_test.go

```diff
@@ -395,29 +395,6 @@ func TestLocalRunOpts_Execute(t *testing.T) {
 			},
 			wantedError: fmt.Errorf("run container: %w", testError),
 		},
-		"successfully run all the containers": {
```

Comment:
why are we removing this testcase 🤔

Reply:
I left that out for now, because it gets timed out. So, there should be a case where we have to handle control c. I tried doing it but its still getting time out. Can you let me know how could we catch control c in this test case?